### ISSUE TITLE
Bug when typing in Korean instead of English in Editor

### ISSUE THREAD

**Issue by hdkhh** [atom/atom#21728]
[Description]
Hello
I am a user of Atom editor in Korea.

There is a bug that occurs when typing in Korean in the editor.
When typing in Korean and clicking the mouse cursor on another line or other place to enter text
It moves to the line where you typed in Korean before and enters it.

This is a very inconvenient bug.
In particular, sometimes there are situations where the previously entered code disappears, such as ctrl + z!
But it doesn't happen when typing in English

I don't know if other foreign languages like Japanese or Chinese have such a bug.
This is the part that needs to be modified.

[Steps to Reproduce]
1. Typing in Korean
2. After completing the input, move the mouse cursor to another line or place.
3. When I type again
4. Go back to the place where I entered, and the last letter is entered
     (Go back to step 1 and type)
5. All the codes I entered in the meantime disappear such as  ctrl+z ...

[Reproduces how often:]
when typing in Korean in the editor.

Versions
1.53.0 x64
Windows 10

**Comment by 2jiwon** [comment]
This happens me a lot. It's just a painful, annoying bug. Especially the code disappearing thing. I am surprised by googling this issue and nobody actually complaining it. Hope this issue would be fixed soon.

**Comment by cspidar** [comment]
I agree. it`s painful.... I assumed that it is just setting problem before checking this issue. I hope there is a solution to fix it.

**Comment by ThatXliner** [comment]
1.55 is the latest version of Atom. Try upgrading Atom first.

**Comment by 2jiwon** [comment]
@ThatXliner yeah, I am using the latest version.
